src/app/cases/page.tsx

```diff
@@ -48,64 +57,67 @@ export default function Page() {
-          options={['Remote/In Person', 'Remote Only', 'In Person Only']}
-          onChange={v =>
-            setCaseFilters({ ...caseFilters, remote: v as string })
-          }
+          defaultValue={defaultFilterValues.remote}
+          multi
+          options={new Set(['Remote', 'In Person'])}
+          value={caseFilters.remote}
+          onChange={v => setCaseFilters({ ...caseFilters, remote: v })}
         />
         <FilterDropdown
-          defaultValue="Attorney/Interpreter"
-          options={['Attorney/Interpreter', 'Interpreter Only']}
-          onChange={v => setCaseFilters({ ...caseFilters, role: v as string })}
+          defaultValue={defaultFilterValues.role}
+          multi
+          options={new Set(['Interpreter', 'Attorney'])}
+          value={caseFilters.role}
+          onChange={v => setCaseFilters({ ...caseFilters, role: v })}
         />
+        {/* <FilterDropdown
+          defaultValue={defaultFilterValues.languages}
+          multi
+          options={new Set(caseData.flatMap(c => c.languages))}
+          value={caseFilters.languages}
+          onChange={v => setCaseFilters({ ...caseFilters, languages: v })}
+        /> */}
         <FilterDropdown
-          defaultValue="Anywhere"
-          // requires knowing user location
-          options={[
-            'Within 5 miles',
-            'Within 10 miles',
-            'Within 20 miles',
-            'Within 100 miles',
-            'Anywhere',
-          ]}
-          onChange={v =>
-            setCaseFilters({ ...caseFilters, distance: v as string })
-          }
+          defaultValue={defaultFilterValues.agency}
+          multi
+          options={new Set(['Court', 'USCIS'])}
+          value={caseFilters.agency}
+          onChange={v => setCaseFilters({ ...caseFilters, agency: v })}
         />
         <FilterDropdown
-          defaultValue="All countries"
+          defaultValue={defaultFilterValues.countries}
           multi
-          options={[
-            'All countries',
-            ...Array.from(
-              new Set(
-                caseData
-                  .filter(c => c.country)
-                  .map(c => (c.country ? c.country : '')),
-              ),
-            ),
-          ]}
-          onChange={v =>
-            setCaseFilters({ ...caseFilters, countries: v as string[] })
+          options={
+            new Set(
+              caseData.filter(c => c.country).map(c => c.country),
+            ) as Set<string>
           }
+          value={caseFilters.countries}
+          onChange={v => setCaseFilters({ ...caseFilters, countries: v })}
         />
       </FiltersContainer>
       <MainDisplay>
         <CardColumn>
           {caseData
             .filter(c => {
-              if (caseFilters.remote === 'remote') return c.is_remote;
-              if (caseFilters.remote === 'inperson') return !c.is_remote;
-              return true;
+              if (caseFilters.remote.size === 0) return true;
```

Comment:
relating to the other comment on this filter: it just feels a bit strange to me that `caseFilters.remote.size === 0` and `caseFilters.remote.size === 2` are essentially the same 🤔

Reply:
I agree, but I think we had this discussion before and I thought the consensus reached was that multiselect all the way was better?

Reply:
yep you're right! i think my worry with the single select filter having one option be plural (i.e. `In Person/Remote`) still stands. but yes this also doesn't feel like an ideal solution, so i definitely see the motivations for having that sort of filter in the first place...let's have a discussion about this during next worksession! cc @davidqing6432